keyboards/planck/keymaps/adamtabrams/rules.mk

```diff
@@ -1,10 +1,9 @@
-SRC += muse.c
 LTO_ENABLE = yes
 
-MOUSEKEY_ENABLE = yes
-EXTRAKEY_ENABLE = yes
-NKRO_ENABLE = yes
-RGBLIGHT_ENABLE = yes
+# MOUSEKEY_ENABLE = yes
+# EXTRAKEY_ENABLE = yes
+# NKRO_ENABLE = yes
+# RGBLIGHT_ENABLE = yes
```

Comment:
If I don't explicitly set these, will they default to what's set in rules.mk for the revision I'm using?
And if so, are those defaults likely to change?
I don't want my config to break just because the rev6/rules.mk file was changed.